### PR TITLE
Bench 309

### PR DESCRIPTION
**Pull request by khayamAmin**

### CODE REVIEW

File: src/Answer.King.Api/appsettings.json [hunk 1]
```diff
@@ -1,16 +1,40 @@
...
+      "WithThreadId"
+    ],
+    "Properties": {
+      "Application": "Bench Project"
     }
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "AnswerKing": "filename=db/Answer.King.db;"
+    "AnswerKing": "filename=Answer.King.db;"
```
Comment (bethcryer) [inline src/Answer.King.Api/appsettings.json]: This was changed from `db/Answer.King.db` by BENCH-305, I think you reverted it in your rebase so could you please change this connection string back to that?

File: src/Answer.King.Api/appsettings.Development.json [hunk 2]
```diff
@@ -1,8 +1,35 @@
...
+      "FromLogContext",
+      "WithMachineName",
+      "WithThreadId"
+    ],
+    "Properties": {
+      "Application": "Bench Project"
     }
+  },
+  "ConnectionStrings": {
+    "AnswerKing": "filename=Answer.King.db;"
```
Comment (bethcryer) [inline src/Answer.King.Api/appsettings.Development.json]: also needs to be `db/Answer.King.db`

**Review by bethcryer — CHANGES_REQUESTED** [review]

**Review by bethcryer — DISMISSED** [review]

**Review by harrystead2 — DISMISSED** [review]

File: src/Answer.King.Domain/Answer.King.Domain.csproj [hunk 3]
```diff
@@ -8,4 +8,20 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="AWSSDK.CloudWatchLogs" Version="3.7.104.4" />
```
Comment (milkshakeuk) [inline src/Answer.King.Domain/Answer.King.Domain.csproj]: wow this doesn't seem right! domain should not reference serilog.

File: src/Answer.King.Infrastructure/Answer.King.Infrastructure.csproj [hunk 4]
```diff
@@ -12,7 +12,20 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="AWSSDK.CloudWatchLogs" Version="3.7.104.4" />
```
Comment (milkshakeuk) [inline src/Answer.King.Infrastructure/Answer.King.Infrastructure.csproj]: pretty sure these are not needed here.

File: src/Answer.King.Api/GlobalSuppressions.cs [hunk 5]
```diff
@@ -0,0 +1,9 @@
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Performance", "CA1848:Use the LoggerMessage delegates", Justification = "Performance improvement is minor", Scope = "member", Target = "~M:Answer.King.Api.Controllers.PaymentsController.GetAll~System.Threading.Tasks.Task{Microsoft.AspNetCore.Mvc.IActionResult}")]
```
Comment (milkshakeuk) [inline src/Answer.King.Api/GlobalSuppressions.cs]: are these supressions needed?

File: .gitignore [hunk 6]
```diff
@@ -462,3 +462,4 @@ $RECYCLE.BIN/
 .terraform.lock.hcl
 terraform.tfstate
 terraform.tfstate.backup
+/src/Answer.King.Api/appsettings.Production.json
```
Comment (bethcryer) [inline .gitignore]: Did you mean to exclude the production appsettings? It's still being tracked in git here

**Review by bethcryer — APPROVED** [review]

**Review by harrystead2 — APPROVED** [review]

**Pull request merged**